go-controller/pkg/ovn/logical_switch_manager/logical_switch_manager.go

```diff
@@ -14,21 +14,21 @@ import (
 	"k8s.io/klog/v2"
 )
 
-// logicalSwitchInfo contains information corresponding to the node. It holds the
+// LogicalSwitchInfo contains information corresponding to the node. It holds the
```

Comment:
does this type need to be public  / exported? I don't see it referenced outside the package.

Reply:
Indeed it doesn't. Will fix it now.